src/sql/Sql.hpp

```diff
@@ -1,10 +1,25 @@
 #pragma once
+#include <vector>
+#include <string>
 
-namespace bakcyl::sql {
+namespace bakcyl::sql 
+{
 
 class Sql {
 public:
     Sql();
+    //Products queries
+    std::vector<Product> selectAllProducts();
+    Product selectProduct(std::uint64_t productId);
+    std::vector<Product> selectProductsBeginningWith(std::string sequence);
```

Comment:
const std::string&, nizej to samo

Typy złożone przekazujmy przez referencję, zeby kompilator nie musiał tutaj tworzyć kosztownej kopii obiektu przy przekazywaniu go do metody